README.md

```diff
@@ -24,11 +24,10 @@ a pre-defined interface (see [extending `itemadapter`](#extending-itemadapter)).
 
 ## Requirements
 
-* Python 3.6+
+* Python 3.7+
 * [`scrapy`](https://scrapy.org/): optional, needed to interact with `scrapy` items
 * `dataclasses` ([stdlib](https://docs.python.org/3/library/dataclasses.html) in Python 3.7+,
```

Comment:
As this is always available in stdlib, it can be removed from requirements.

Reply:
Done!